src/test/java/ua/lviv/iot/AppTest.java

```diff
@@ -0,0 +1,20 @@
+//package ua.lviv.iot;
+//
+//import static org.junit.Assert.assertTrue;
+//
+//import org.junit.Test;
+//
+///**
+// * Unit test for simple App.
+// */
+//public class AppTest

```

Comment:
Yeah... Well... Why would you want such a file in your source code?

Reply:
Deleted